indigo_api/models/documents.py

```diff
@@ -548,6 +548,15 @@ def change_date(self, new_date, user, comment=None):
             self.expression_date = new_date
             self.save_with_revision(user, comment=comment)
 
+    def get_provision_xml(self, provision_eid):
+        provision_xml = self.doc.get_portion_element(provision_eid)
+        if not provision_xml:
```

Comment:
```suggestion
        if provision_xml is None:
```

lxml spits out a warning when treating elements as booleans